src/layouts/BaseLayout.astro

```diff
@@ -11,10 +11,13 @@ const { pageTitle } = Astro.props;
 
     <!-- CSS only -->
     <link rel="stylesheet" href="https://assets.emblstatic.net/vf/v2.5.12/css/styles.css">
+    <link rel="stylesheet" href="https://cdn.datatables.net/2.0.8/css/dataTables.dataTables.css" />
+
     <!-- JS -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+    <script src="https://cdn.datatables.net/2.0.8/js/dataTables.js"></script>
```

Comment:
Is this ever used? Looks like it's a different version than https://cdn.datatables.net/1.11.5/js/jquery.dataTables.js